neo/test/rawiotest/test_spikeglxrawio.py

```diff
@@ -85,6 +86,20 @@ def test_subset_with_sync(self):
         )
         assert chunk.shape[1] == 120
 
+    def test_nidq_digital_channel(self):
+        rawio_digital = SpikeGLXRawIO("spikeglx/DigitalChannelTest_g0")
+        rawio_digital_channel.parse_header()
```

Comment:
```suggestion
        rawio_digital.parse_header()
```
This was just a typo.

Reply:
Good call! Let's see ;)